copied-deps/eastwood/copieddeps/dep2/clojure/tools/analyzer/jvm.clj

```diff
@@ -582,9 +583,23 @@
               :raw-forms  raw-forms})
            (let [a (analyze mform env opts)
                  frm (emit-form a)
-                 result (try (eval frm) ;; eval the emitted form rather than directly the form to avoid double macroexpansion
-                             (catch Exception e
-                               (handle-evaluation-exception (ExceptionThrown. e a))))]
+                 result (try
```

Comment:
I think it would be prudent (if possible) to signal in some way that this code is monkey-patched. Imagine if we were to bump t.a.j, in the future, we would most likely miss this bit here.

Reply:
Agreed, done!